### PR TITLE
handle .tar.bz2 and .tar.lzma files

### PR DESCRIPTION
**Pull request by mmoll**
see rcrowley/freight#61

### CODE REVIEW

File: lib/freight/apt.sh [hunk 1]
```diff
@@ -112,7 +112,7 @@ apt_cache() {
 			# and will find the associated *.orig.tar.gz, *.diff.gz, and/or
             # *.tar.gz as they are needed.
 			*.dsc) apt_cache_source "$DIST" "$DISTCACHE" "$PATHNAME" "$COMP" "$PACKAGE";;
-			*.debian.tar.gz|*.debian.tar.xz|*.diff.gz|*.orig.tar.gz|*.orig.tar.xz|*.tar.gz|*.tar.xz|*.deb-control|*.dsc-cached) ;;
+			*.debian.tar.gz|*.debian.tar.bz2|*.debian.tar.xz|*.diff.gz|*.orig.tar.gz|*.orig.tar.bz2|*.orig.tar.xz|*.orig.tar.lzma|*.tar.gz|*.tar.bz2|*.tar.xz|*.deb-control|*.dsc-cached) ;;
```
Comment (domcleal) [inline lib/freight/apt.sh]: missing `*.debian.tar.lzma` and `*.tar.lzma` ?

Comment (mmoll) [inline lib/freight/apt.sh]: indeed

**Comment by domcleal** [comment]
:+1: looks good, thanks.

**Comment by mattock** [comment]
This look good afaics.

**Pull request merged**